nodebuilder/tests/api_test.go

```diff
@@ -168,3 +182,35 @@ func TestHeaderSubscription(t *testing.T) {
 	err = light.Stop(ctx)
 	require.NoError(t, err)
 }
+
+func TestSubmitBlobOverHTTP(t *testing.T) {
```

Comment:
This is the same test as  TestBlobRPC, but without usage of client lib, instead it uses raw json request. The downside of raw api tests is that every time api is changed they would need to be manually updated.

Reply:
That's the point, we want this (for every method though)

Reply:
Approved bacause it is fine as long as there not many tests like that. Because it may become burden to maintain such tests. For future API work (Canonical api) we should rather do backwards compatibility tests